.pre-commit-config.yaml

```diff
@@ -1,10 +1,23 @@
 ---
-fail_fast: false
-default_stages:
-  - pre-commit
-  - pre-push
-
+# https://pre-commit.com
+default_install_hook_types: [commit-msg, pre-commit]
+default_stages: [pre-commit, manual]
+fail_fast: true
```

Comment:
Why change `fail_fast` to `true`? Isn't this better to see everything wrong rather than fixing one issue at a time if there's more than one?